cli/ts/commands/deployPoll.ts

```diff
@@ -196,10 +147,22 @@ export const deployPoll = async (
     const pollId = log!.args._pollId as number;
     // eslint-disable-next-line no-underscore-dangle
     pollAddr = log!.args._pollAddr as string;
+    // eslint-disable-next-line no-underscore-dangle
+    messageProcessorContractAddress = log!.args._mpAddr as string;
+    // eslint-disable-next-line no-underscore-dangle
+    tallyContractAddress = log!.args._tallyAddr as string;
+
+    const receiptLogSubsidy = receipt!.logs[receipt!.logs.length - 2];
```

Comment:
As above, there should be a flag and then here try to get address only if the user decided to deploy it